src/App.tsx

```diff
@@ -1,27 +1,118 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import {
+  Container,
+  Stack,
+  Button,
+  Typography,
+} from '@mui/material';
+
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
+
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const [
+    allButtonVariant,
+    setAllButtonVariant,
+  ] = useState<'outlined' | 'contained'>('outlined');
+
+  const [
+    firstFiveButtonVariant,
+    setFirstFiveButtonVariant,
+  ] = useState<'outlined' | 'contained'>('outlined');
+
+  const [
+    redButtonVariant,
+    setRedButtonVariant,
+  ] = useState<'outlined' | 'contained'>('outlined');
+
+  const handleLoadAll = async () => {
+    if (allButtonVariant === 'outlined') {
+      setGoods(await getAll());
+      setAllButtonVariant('contained');
+      setFirstFiveButtonVariant('outlined');
+      setRedButtonVariant('outlined');
+    } else {
+      setGoods([]);
+      setAllButtonVariant('outlined');
+    }
```

Comment:
I think if we invert condition it will improve code readability(optional change)
```suggestion
     if (allButtonVariant !== 'outlined') {
       setGoods([]);
       setAllButtonVariant('outlined');
       return
     } 
    
      setGoods(await getAll());
      setAllButtonVariant('contained');
      setFirstFiveButtonVariant('outlined');
      setRedButtonVariant('outlined');
```

Reply:
@dffUqp it's awesome, thanks